framework/rbac/DbManager.php

```diff
@@ -753,6 +754,24 @@ public function hasChild($parent, $child)
     /**
      * @inheritdoc
      */
+    public function getParents($name)
```

Comment:
may be add second argument `getParents($name, $type = Item::TYPE_ROLE)` and add condition for select?